chain/sync.go

```diff
@@ -1187,13 +1196,31 @@ func (syncer *Syncer) iterFullTipsets(ctx context.Context, headers []*types.TipS
 	defer span.End()
 
 	span.AddAttributes(trace.Int64Attribute("num_headers", int64(len(headers))))
+	ss := extractSyncState(ctx)
 
 	windowSize := 200
 	for i := len(headers) - 1; i >= 0; {
-		fts, err := syncer.store.TryFillTipSet(headers[i])
+
```

Comment:
@whyrusleeping @magik6k This won't do anything weird, yeah?

Reply:
(by "this", i mean using the cache to skip a `TryFillTipset`)

Reply:
Yeah, It should be fine. If we've validated it, that means we had to have had all the messages in it anyways.

That said, we should probably be doing this on a per block basis